### PR TITLE
Fix database commands.

### PR DESCRIPTION
**Pull request by HarmanPC**
Please do not merge this PR yet, I'm fixing delete etc., too.

### CODE REVIEW

**Comment by HarmanPC** [comment]
Okay, I'm done with commits.

File: plugins/writing.js [hunk 1]
```diff
@@ -2100,8 +2101,22 @@ let commands = {
 			database.myths.pending = input;
 			Storage.exportDatabase('writing');
 			return this.say("To confirm addition of ``" + input.name + "`` under pantheon ``" + input.pan + "``, type ``;myth confirm, add``.");
-		} else if (targets[0] === "confirm") {
+		}	else if (targets[0] === "remove" || targets[0] === "delete") {
```
Comment (sirDonovan) [inline plugins/writing.js]: There should only be 1 space between <code>{</code> and <code>else</code>.

Comment (sirDonovan) [reply plugins/writing.js]: Also can you put this back under the <code>addimage</code> block so it's easier to review what was actually changed?

**Comment by HarmanPC** [comment]
Oh, sorry I'll change, I used tab x.x

**Comment by HarmanPC** [comment]
I actually put it up because: add->delete->confirm was reducing confusions.

**Pull request closed without merging**